hagglehaul.Server/Controllers/RiderController.cs

```diff
@@ -51,6 +48,114 @@ public async Task<IActionResult> Get()
             return Ok(riderBasicInfo);
         }
 
+        [Authorize]
+        [HttpGet]
+        [Route("dashboard")]
+        [ProducesResponseType(typeof(RiderDashboard), StatusCodes.Status200OK)]
+        public async Task<IActionResult> GetDashboard()
```

Comment:
Needs attributes 
Needs: 
* Trips in bidding: per bid:  driver name, driver rating, driver number of ratings

Yours: 
```json
{
  "confirmedTrips": [
    {
      "tripID": "string",
      "tripName": "string",
      "thumbnail": "string",
      "startTime": "2024-02-26T08:55:03.040Z",
      "distance": 0,
      "duration": 0,
      "cost": 0,
      "riderName": "string",
      "riderRating": 0,
      "riderNumRating": 0,
      "showRatingPrompt": true,
      "riderEmail": "string",
      "riderPhone": "string",
      "pickupAddress": "string",
      "destinationAddress": "string"
    }
  ],
  "tripsInBidding": [
    {
      "tripID": "string",
      "tripName": "string",
      "thumbnail": "string",
      "startTime": "2024-02-26T08:55:03.040Z",
      "distance": 0,
      "duration": 0,
      "riderName": "string",
      "riderRating": 0,
      "riderNumRating": 0,
      "bids": [
        {
          "id": "string",
          "driverEmail": "string",
          "tripId": "string",
          "centsAmount": 0
        }
      ],
      "pickupAddress": "string",
      "destinationAddress": "string"
    }
  ],
  "archivedTrips": [
    {
      "tripId": "string",
      "tripName": "string",
      "thumbnail": "string",
      "startTime": "2024-02-26T08:55:03.040Z",
      "distance": 0,
      "duration": 0,
      "cost": 0,
      "riderName": "string",
      "riderRating": 0,
      "riderNumRating": 0,
      "pickupAddress": "string",
      "destinationAddress": "string"
    }
  ]
}
```